Readme.md

```diff
@@ -1,15 +1,21 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128578657/22.2.2%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2122)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+
+# Pivot Grid for WPF - Hide Column Totals
+
+This example demonstrates how to handle the [CustomCellValue](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.PivotGridControl.CustomCellValue?p=netframework) event to hide the _Discount_ field column grand totals. 
```

Comment:
`field column grand totals` This doesn't make sense. Can you explain?

Reply:
```suggestion
This example demonstrates how to handle the [CustomCellValue](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.PivotGridControl.CustomCellValue?p=netframework) event to hide the _Discount_ field grand totals. 
```